src/utils.h

```diff
@@ -112,6 +112,18 @@ namespace Utils {
 	}
 
 	/**
+	 * Calculates the modulo of number i ensuring the result is positive for
```

Comment:
Technically, it's only non-negative (i=0).

Even more techincally, m has to be >0, or else eg. `PositiveMod(1,-2)` is -1.